bin/provision_runner

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if ! which apt >/dev/null; then
+  echo "Requires debian-like system"
+fi
+
+sudo apt-get update
+sudo apt-get install -y \
+  git build-essential libtool autotools-dev automake \
+  pkg-config libssl-dev libevent-dev bsdmainutils ccache libqt5gui5 \
+  libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev \
+  protobuf-compiler libboost-system-dev libboost-filesystem-dev \
+  libboost-chrono-dev libboost-program-options-dev libboost-test-dev \
+  libboost-thread-dev \
```

Comment:
Missing `clang` (see runner/Dockerfile)?

Could you somehow use this in the runner/Dockerfile, so it is not duplicated?

Reply:
Ah, good catch. Willfix the docker stuff in an upcoming PR.